new-site/docs-site/content/0.19.0/guide/typesense-dynamodb.md

```diff
@@ -0,0 +1,240 @@
+# DynamoDB Integration with Typesense
+
+Hey there! This post give you a guide to integrate Typesense cluster with AWS DynamoDB by setting up a trigger with AWS Lambda.
+
+![Typesense DynamoDB Integration Chart](~@guide-images/typesense-dynamodb/typesense-dynamodb.svg)
+
+## Step 1: Create Typesense Cluster
+
+Sign up for a new account in [Typesense Cloud](https://cloud.typesense.org/) and get Endpoint URL, Port number and API key.
+## Step 2: Create a DynamoDB table
+
+Create a DynamoDB table with your choice of name and partition key ("id" is recommended).
+
+After creating a DynamoDB table you want to enable streams in the "Overview" section.
+
+You can also do this using AWS CLI:
+
+```bash
+aws dynamodb create-table \
+    --table-name typensense \
```

Comment:
Could you change the table name to something else? May be `YourTableName` or something